### ISSUE TITLE
Can not load created ModPacks in the Curse launcher 

### ISSUE THREAD

**Issue by nmarshall23** [dizzyd/mcdex#20]
Modpacks created by mcdex can't be loaded in the Twitch launcher. This is because how mcdex lists forge versions with just the version in the id string. 

The Twitch launcher requires the prefix 'forge' then a dash then the version number.

Json partial created by mcdex using `pack.create`.

```json
{
"minecraft": {
  "modLoaders": [
   {
    "id": "14.23.1.2575",
    "primary": true
   }
  ]
}
```

The same json partial created by the Twitch launcher, when exporting a modpack.
```json
{
"minecraft": {
  "modLoaders": [
   {
    "id": "forge-14.23.1.2575",
    "primary": true
   }
  ]
}
```

**Comment by dizzyd** [comment]
Thanks for the report @nmarshall23. I've tweaked the pack.create functionality to always prefix forge-; can you let me know if there are any other alterations you need, please?

**Comment by dizzyd** [comment]
Fixed in v0.8.4.
